homeassistant/components/filesize/coordinator.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, hass: HomeAssistant, unresolved_path: str) -> None:
             always_update=False,
         )
         self._unresolved_path = unresolved_path
-        self._path: pathlib.Path | None = None
+        self.path: pathlib.Path = None  # type: ignore[assignment]
```

Comment:
instead, this can be defined as a class variable, that isn't assigned a `None` value.